### PR TITLE
Fix for service preferences.

### PR DESCRIPTION
**Pull request by eozan**

### CODE REVIEW

File: client/app/index.html [hunk 1]
```diff
@@ -171,7 +171,7 @@
     <script src="node_modules/superdesk-core/scripts/superdesk-authoring/versioning/versioning.js"></script>
     <script src="node_modules/superdesk-core/scripts/superdesk-authoring/versioning/versions/versions.js"></script>
     <script src="node_modules/superdesk-core/scripts/superdesk-authoring/versioning/history/history.js"></script>
-    <script src="node_modules/superdesk-core/scripts/superdesk-authoring/metadata/metadata.js"></script>
+    <script src="scripts/superdesk-authoring/metadata/metadata.js"></script>
```
Comment (petrjasek) [inline client/app/index.html]: this should be probably `app/scripts/...`

Comment (petrjasek) [inline client/app/index.html]: or does it work locally for you?

Comment (petrjasek) [inline client/app/index.html]: when using grunt build?

Comment (eozan) [inline client/app/index.html]: it works locally for me, but changed the path to app/scripts/..., and it still works, grunt build gives no errors.

Comment (petrjasek) [inline client/app/index.html]: it won't return error, warning maybe. so does it work?

**Pull request closed without merging**